index.html

```diff
@@ -20,13 +21,16 @@
       </div>
       <div> 
         <ul class="menu">
+
           <a href="#"><li>Portfolio</li></a>
           <a href="#"><li>About</li></a>
           <a href="#"><li>Contact</li></a>
```

Comment:
- [ ] Kindly ensure that a proper portion of the page is shown when the user clicks on any of the mobile items on the menu.
Please check [this](https://blog.hubspot.com/marketing/jump-link-same-page) link for the know-how 😉.